packages/x-components/static-docs/experience-search-and-discovery/README.md

```diff
@@ -130,3 +103,29 @@ your native app search solution.
     'x_empathize',
     'x_results',]"
 />
+
+::: warning
+
+Some X&nbsp;Components are supported by Empathy Platform search and discovery features. If you’re
+using a different search engine, make sure it supports all the functionalities you need.
+
+:::
+
+## Try X Components to…
+
+- Surprise your shoppers by **rendering shopping-aimed search and discovery features** that
+  intuitively guide and inspire them with trends, suggestions, and recommendations, based on your
+  shoppers’ behavior.
```

Comment:
```suggestion
  intuitively guide and inspire them with trends, suggestions, and recommendations, based on their behavior.
```

Reply:
Both options are correct. But using continuously possessive pronouns/adjectives can make your idea imprecise. Specifying the actor prevents readers from doubting.